doc/src/quickstart.md

```diff
@@ -0,0 +1,91 @@
+# Quickstart Guide
+
+This is a short guide to get you started. For a more detailed explanation about
+what's going on, please refer to the [tutorial](tutorial/index.html).
+
+We begin by creating a `Cargo.toml` file and load the necessary dependencies:
+
+```toml
+[package]
+name = "project"
+version = "0.0.1"
+authors = ["<authors>"]
+
+[[bin]]
+doc = false
+name = "project"
+
+[build-dependencies]
+lrlex = { path="<path to lrlex>" }
+lrpar = { path="<path to lrpar>" }
+
+[dependencies]
+cfgrammar = { path="<path to cfgrammar>" }
+lrlex = { path="<path to lrlex>" }
+lrpar = { path="<path to lrpar>" }
+```
+
+In the root of the project we create a file `build.rs`, which processes our
+lexer and grammar files. Here we can choose whether we want to generate a
+default parse tree, or use grammar actions instead:
+
+```rust
+extern crate lrlex;
+extern crate lrpar;
+
+use lrlex::LexerBuilder;
+use lrpar::{CTParserBuilder, ActionKind};
+
+fn main() -> Result<(), Box<std::error::Error>> {
+    let lex_rule_ids_map = CTParserBuilder::new().action_kind(ActionKind::CustomAction).process_file_in_src("project.y")?;
+    LexerBuilder::new()
+        .rule_ids_map(lex_rule_ids_map)
+        .process_file_in_src("project.l")?;
+    Ok(())
+}
+```
+
+Now we can create the main function for our program which uses the parser
+generated in `build.rs`. When run this problem will take user input, lex and
+parse it according to the grammar, and return the result of the grammar
+actions:
+
+```rust
+extern crate lrpar;
+
+use lrpar::{LexParseError, Lexer};
+
+lrlex_mod!(project_l);
```

Comment:
This stuff needs to be commented otherwise it's just gobbledegook. `lrpar/examples/calcparse`'s `build.rs` and `src/main.rs` might be commented well enough?

Reply:
I removed the comments, because I wanted the quickstart guide to be concise and move the explanations to the tutorial. But I will put the comments back in.